fdbserver/workloads/WriteTagThrottling.actor.cpp

```diff
@@ -0,0 +1,228 @@
+/*
```

Comment:
My approach to this (which I haven't committed, but I do have as an outstanding change) was to update the `ReadWrite` workload to support applying tags to transactions in targeted ways. I haven't thoroughly compared this workload with the existing `ReadWrite` one, but if they are substantially similar, we may want to consider doing that or some variant of it.